spec/services/cache_service_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe CacheService do
+  let(:rails_cache) { Rails.cache }
+
+  describe "#cache" do
+    before do
+      rails_cache.stub(:fetch)
```

Comment:
this syntax is from the old past. We should use `allow()`: https://relishapp.com/rspec/rspec-mocks/v/3-9/docs/basics/allowing-messages

Reply:
Updated :+1: